pkg/dockerutil/dockerutils.go

```diff
@@ -793,21 +810,18 @@ func GetContainerEnv(key string, container dockerTypes.Container) string {
 	return ""
 }
 
-// CheckDockerVersion determines if the Docker version of the host system meets the provided version
-// constraints. See https://godoc.org/github.com/Masterminds/semver#hdr-Checking_Version_Constraints
-// for examples defining version constraints.
-func CheckDockerVersion(versionConstraint string) error {
+// CheckDockerVersion determines if the Docker version of the host system meets the provided
+// minimum for the Docker API Version.
+func CheckDockerVersion(dockerVersionMatrix DockerVersionMatrix) error {
 	defer util.TimeTrack()()
 
 	currentVersion, err := GetDockerVersion()
 	if err != nil {
 		return fmt.Errorf("no docker")
 	}
-	// If Docker version has "_ce", remove it. This happens on OpenSUSE Tumbleweed at least
-	currentVersion = strings.TrimSuffix(currentVersion, "_ce")
-	dockerVersion, err := semver.NewVersion(currentVersion)
+	currentAPIVersion, err := GetDockerAPIVersion()
 	if err != nil {
-		return err
+		return fmt.Errorf("no docker")
```

Comment:
This could certainly be improved with an actual error type, but no need to change. It's been this way a long time. But we should really be doing fancier.